config/locales/en/activemodel.yml

```diff
@@ -41,8 +41,8 @@ en:
         dob_year: Year
         name: Name
       gateway_evidence:
-        file_uploaded: "You have uploaded %{file_name}"
-        file_deleted: "You have deleted %{file_name}"
+        file_uploaded: "%{file_name} has been successfully uploaded."
```

Comment:
Do these need to be changed, the reference now seems to  point to line 71?

Reply:
Ah yes good point I will remove these